tests/FSharp.Compiler.Service.Tests/ProjectAnalysisTests.fs

```diff
@@ -4645,7 +4645,7 @@ let callToOverload = B(5).Overload(4)
     let args = mkProjectCommandLineArgs (dllName, [])
 
 [<Theory>]
-// [<InlineData true>] // Flaky, reenable when stable
+[<InlineData true>] // Flaky, reenable when stable
```

Comment:
If these were flaky before, is there a reason to expect that to improve now? Anyway, will be good to see what goes wrong here if anything.

Reply:
Right, it definitely became stable in the context of parallel tests experiment. If some instability appears, l would suspect insufficient isolation of test cases.